Support/gdbtrace.init

```diff
@@ -839,9 +839,32 @@ define _doTRACE
   set *($TPIUBASE+0x304) = 0x102
 end
 # ====================================================================
-define enableSTM32TRACE
+define _configSTM32Pin
+  #set language c
+
+  # MODER: Alternate Function
+  set *0x40021000 &= ~(0x3<<2*$arg0)
+  set *0x40021000 |=  (0x2<<2*$arg0)
+  # OTYPER: Push-Pull
+  set *0x40021004 &= ~(0x1<<$arg0)
+  # OSPEEDR: Drive speed
+  set *0x40021008 &= ~(0x3<<2*$arg0)
+  set *0x40021008 |=  ($arg1<<2*$arg0)
+  # PUPDR: No pullup or pulldown
+  set *0x4002100C &= ~(0x3<<2*$arg0)
+  # AFRL: AF0
+  set *0x40021020 &= ~(0xF<<4*$arg0)
+  # LOCKR: lock pin until the next reset
+  set *0x4002101C = 0x10000 | (0x1<<$arg0)
+  set *0x4002101C = 0x00000 | (0x1<<$arg0)
+  set *0x4002101C = 0x10000 | (0x1<<$arg0)
+  set $_null = *0x4002101C
```

Comment:
Is there a more elegant way to silently read a memory location in GDB? (ie. not with x)